tools/junitreport/pkg/parser/gotest/example/example_test.go

```diff
@@ -0,0 +1,40 @@
+package example
+
+/* Used to generate output for testing
```

Comment:
I'm assuming we're commenting these out not so that go does not pick it up. In that case why not just turn this into .txt file or alike? I hate big chunks of commented out code, it looks weird.